backend/server.py

```diff
@@ -45,77 +45,86 @@
 data = [
 {
     "id": 1,
-    "jobCategory": "Project Management",
-    "jobCategoryDescription": "Delivering exciting, innovative, complex, and technical projects.",
+    "jobCategory": "IT, Software & Digital",
```

Comment:
so the overall strucutre still same with main 4 department, then follow by few available job under each department?